modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -691,7 +691,8 @@ public class IgniteImpl implements Ignite {
                 name,
                 metastorageWorkDir.dbPath(),
                 failureManager,
-                readOperationForCompactionTracker
+                readOperationForCompactionTracker,
+                threadPoolsManager.commonScheduler()
```

Comment:
`threadPoolsManager.commonScheduler()` should only be used for light, fast tasks, never doing I/O, but here it's used for I/O, as it seems. Let's use another pool here

Reply:
I think that it is necessary to fix the fact that IO operations take place at `RocksDbFlusher#scheduledPool`.